code/routers/user/donation.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express')
+const router = express.Router();
+
+const Donation = require('../../models/donation');
+
+const donationCodes = require('../../config/resCodes').donation;
+
+/* 
+	Donation API:
+		Create new donation:
+			- Input: donation info, newTripID, and a callback function
+			- Create new mongoose donation object with given info
+			- Save to database
+			- Call callback with either success or fail information
```

Comment:
Take a look at how the other functions are being commented and try to follow that format. Also since this isn't an API "Donation API" should be removed since it is a bit misleading.